packages/astro/src/@types/astro.ts

```diff
@@ -469,6 +469,17 @@ export interface AstroUserConfig {
 	 */
 	trailingSlash?: 'always' | 'never' | 'ignore';
 
+	/**
+	 * @docs
+	 * @name prependDoctype
+	 * @type {boolean}
+	 * @default true
```

Comment:
For better rendering in docs, we wrap default values with backticks.
```suggestion
	 * @default `true`
```